edx_sga/tests/integration_tests.py

```diff
@@ -779,8 +780,10 @@ def test_replace_url(self, has_static_asset_path):
         If the static asset path is set on a course, it should be substituted when the course is rendered
         """
         # make a runtime with a static asset path, which will override the base_asset_url
-
-        static_asset_path = '/a/different/static/asset/path'
+        if django.VERSION >= (1, 11):
+            static_asset_path = '/static/a/different/static/asset/path'
+        else:
+            static_asset_path = '/a/different/static/asset/path'
```

Comment:
My understanding is that this `static_asset_path` setting is testing a particular case, and the existing value is arbitrary. 

How did django 1.11 change that the old, arbitrary path no longer work? 

Can we just change it for all versions of django, to be `static_asset_path = '/static/a/different/static/asset/path'` ?